components/feature/addons/src/main/res/values-ne-rNP/strings.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources xmlns:tools="http://schemas.android.com/tools">
+    <!-- Description for privacy add-on permission. -->
+    <string name="mozac_feature_addons_permissions_privacy_description">पढ्नुहोस् र गोपनीयता सेटिंग्स परिमार्जन गर्नुहोस्</string>
+    <!-- Description for all_urls add-on permission. -->
+    <string name="mozac_feature_addons_permissions_all_urls_description">सबै वेबसाइटहरूका लागि तपाईंको डाटा पहुँच गर्नुहोस्</string>
+    <!-- Description for giving an add-on access to users's data on one site. %1$s  will be replaced by the DNS host name for which a web extension is requesting access (e.g., www.mozilla.org). -->
+    <string name="mozac_feature_addons_permissions_one_site_description">%1$ s को लागि तपाईंको डाटा पहुँच गर्नुहोस्</string>
```

Comment:
@Delphine I think this should be %1$s ?  :)

Reply:
Gha! yes, it should, thanks :) 
I will reach out once more to localizers. Many are not technical and so sometimes they don't realize this

Reply:
Updated on Pontoon, should be good to know now